TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/AssetController.php

```diff
@@ -255,4 +258,25 @@ protected function getErrorFlashMessage()
         }
         return new Error($errorMessage, null, [get_class($this), $this->actionMethodName]);
     }
+
+    /**
+     * Add a translated flashMessage.
+     *
+     * @param string $messageBody The translation id for the message body.
+     * @param string $messageTitle The translation id for the message title.
+     * @param string $severity
+     * @param array $messageArguments
+     * @param integer $messageCode
+     * @return void
+     */
+    public function addFlashMessage($messageBody, $messageTitle = '', $severity = Message::SEVERITY_OK, array $messageArguments = array(), $messageCode = null)
+    {
+        $localeObject = $this->_localizationService->getConfiguration()->getCurrentLocale();
```

Comment:
Do you mind to elaborate what you mean then? Because if I check that trait and how it is used it's not clear from the code what magic is happening and the trait itself does not seem to introduce any translate methods, and the only docs are "A trait to add backend translation based on the backend users settings" which is not really self explaining too.

Is it possible to leave out the Locale object just because of that code in the initializeObject() of the trait?